extensions/wikia/Email/templates/wallMessage.mustache

```diff
@@ -0,0 +1,49 @@
+<div class="greeting">
+	<p>{{salutation}}</p>
+	<p>{{{summary}}}</p>
+</div>
+
+<table>
+	<tr>
+		<td valign="top">
+			<div class="avatar-wrapper">
+				{{# editorProfilePage}}
+					<a href="{{editorProfilePage}}">
+				{{/ editorProfilePage}}
+					<img class="avatar" src="{{editorAvatarURL}}">
+				{{# editorProfilePage}}
+					</a>
+				{{/ editorProfilePage}}
+			</div>
+		</td>
+		<td class="widest">
+			<div class="message">
+				<p>
+					{{# editorProfilePage}}
+						<a class="username" href="{{editorProfilePage}}">
+					{{/ editorProfilePage}}
+						{{editorUserName}}
+					{{# editorProfilePage}}
+						</a>
+					{{/ editorProfilePage}}
+				</p>
+				<strong>{{wallMessageTitle}}</strong><br><br>
+				<p>{{wallMessageBody}}</p>
```

Comment:
Hmm, this is a ton of duplicated code. The only real difference between this file and avatarLayout.mustache that I can see are L30-31. What we could do is put the contents of those 2 lines into the the `{{details}}` variable [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/templates/avatarLayout.mustache#L30) inside of avatarLayout. That way we can just reuse the one template.

Also, I feel like we shouldn't be using the `<br>` tags. Seems like something we should be doing with CSS? The wallMessageTitle should probably be in a `<p>` tag or something anyways, yea?